src/main/java/Script.java

```diff
@@ -0,0 +1,107 @@
+import org.json.JSONArray;
+import java.util.Map;
+/**
+ * @author B. Clint Hall
+ * @version 1
+ */
+public class Script {
```

Comment:
@bclinthall  Could you explain the logic of making most of this class' functionality static? I don't understand.